docker-compose.yml

```diff
@@ -28,5 +28,7 @@ services:
       - S3_RTR_PUBLIC=s3-anonymous://mbta-gtfs-s3/
       - GLIDES_PERSONNEL_PATHNAME=personnel/demo.csv
       - GLIDES_RFID_PATHNAME=rfid/demo.csv
+      - OCS_KINSESIS_STREAM_NAME=ctd-ocs-raw-messages
```

Comment:
nit: typo

Reply:
I would say this is not a nit. I did not proliferate the typo to the infra work, so this would have actually caused a problem 🤦